Akkadu/rsi-api-widgets
# Feature/control player page

Please describe the purpose of the pull request.

* [notion page](https://www.notion.so/akkadu/RSI-API-pause-play-programmatically-the-VM-player-49e7785ec3834236a09d4f5ad1803f3f)
* For the interpretation player we want to be able to give the option to play/pause programmatically the player of the virtual platforms.


## How To Test



* Go on this page : https://deploy-preview-2--rsi-akkadu-vue-demo.netlify.app/#/and start the player,
* When selecting the `interpreted language` (here chinese), it should `mute` the video
* when selecting the `floor language` (english) it shoud `unmute` the video.

## Screenshots (if appropriate)

## Checklist



加油！
